libr/include/r_anal.h

```diff
@@ -1316,6 +1316,10 @@ static inline bool r_anal_block_contains(RAnalBlock *bb, ut64 addr) {
 // The returned block will always be refd, i.e. it is necessary to always call r_anal_block_unref() on the return value!
 R_API RAnalBlock *r_anal_block_split(RAnalBlock *bb, ut64 addr);
 
+static inline bool r_anal_block_is_contiguous(RAnalBlock *a, RAnalBlock *b) {
+	return a->addr + a->size == b->addr;
```

Comment:
add parenthesis to make clear you are not comparing the size, but the result of the addition